components/table/table-wrapper.js

```diff
@@ -124,9 +124,12 @@ export const tableStyles = css`
 			--d2l-table-col-sort-button-width: unset;
 		}
 		/* has at least one d2l-table-col-sort-button with [nosort], does not have d2l-table-col-sort-button without nosort */
-		.d2l-table > * > tr > :has(d2l-table-col-sort-button[nosort]:not(:only-child)):not(:has(d2l-table-col-sort-button:not([nosort]))) :last-child {
+		.d2l-table > * > tr > * > d2l-table-col-sort-button[nosort] ~ :last-child {
```

Comment:
I just tried to keep the specificity the same, but I'm wondering if we can/should do this instead:
```suggestion
		.d2l-table > * > th d2l-table-col-sort-button[nosort] ~ :last-child {
```

Reply:
That should be safe. We're not expecting sort buttons outside of the `thead` I presume? And the updated CSS would target nested tables but again probably not a concern inside header cells.